tests/contracts/solcontract_2.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
```

Comment:
Added a second contract for the sake of testing that multiple contracts don't all have the same `sourceID` (that was the bug that was happening). You can see in the test I am asserting that the `sourceId` is expected now.